subprojects/testing-base/src/main/java/org/gradle/api/tasks/testing/AbstractTestTask.java

```diff
@@ -551,7 +551,7 @@ private void createReporting(Map<String, TestClassResult> results, TestOutputSto
      *
      * For more information on supported patterns see {@link TestFilter}
      */
-    @Option(option = "tests", description = "Sets test class or method name to be included, '*' is supported.")
+    @Option(option = "tests", description = "Sets test class or method name to be included (in addition to build-script definitions), '*' is supported.")
```

Comment:
I think we can make this even clearer. What about this?
```suggestion
    @Option(option = "tests", description = "Sets test class or method name to be included (in addition to the test task filters), '*' is supported.")
```